src/components/offer-card-square/OfferCardSquare.tsx

```diff
@@ -0,0 +1,81 @@
+import Box from '@mui/material/Box'
+import Typography from '@mui/material/Typography'
+import LanguageIcon from '@mui/icons-material/Language'
+import IconButton from '@mui/material/IconButton'
+import TurnedInNot from '@mui/icons-material/TurnedInNot'
+
+import { t } from 'i18next'
+import AppCard from '~/components/app-card/AppCard'
+import ImgTitleDescription from '~/components/img-title-description/ImgTitleDescription'
+import AppChip from '../app-chip/AppChip'
+import AppRatingMobile from '../app-rating-mobile/AppRatingMobile'
+import AppButton from '~/components/app-button/AppButton'
+
+import { OfferCard } from '~/types'
+
+import { styles } from '~/components/offer-card-square/OfferCardSquare.styles'
+
+interface OfferCardSquareProps {
+  offer: OfferCard
+}
+
+const OfferCardSquare: React.FC<OfferCardSquareProps> = ({ offer }) => {
+  const { bio, photo, firstName, lastName, languages, subject, proficiencyLevel, price, averageRating, totalReviews } =
+    offer
+
+  const levelText =
+    proficiencyLevel === 'Beginner'
+      ? t('common.beginner')
+      : `${t('common.beginner')} - ${proficiencyLevel}`.toUpperCase()
+
+  return (
+    <AppCard isClickable={ false } sx={ styles.containerCard }>
+      <Box sx={ styles.container }>
+        <ImgTitleDescription
+          description={ bio } img={ photo } style={ styles.mainInfo }
+          title={ `${firstName} ${lastName}` }
+        />
+        <IconButton sx={ styles.iconButton }>
+          <TurnedInNot />
+        </IconButton>
+        <Box sx={ styles.languagesContainer }>
+          <LanguageIcon sx={ styles.languageIcon } />
+          <Typography sx={ styles.languages } variant='body2'>
+            { languages.join(', ') }
+          </Typography>
+        </Box>
+        <Box sx={ styles.chipsContainer }>
+          <AppChip labelSx={ styles.subjectChipLabel } sx={ styles.subjectChip }>
+            { subject.toUpperCase() }
+          </AppChip>
+          <AppChip labelSx={ styles.levelChipLabel } sx={ styles.levelChip }>
+            { levelText }
+          </AppChip>
+        </Box>
+
+        <Box sx={ styles.priceContainer }>
```

Comment:
As i see we are using this block with price in some other places, better to create a reusable component

Reply:
fixed